eth_verifier/lib/bn254/Fields.sol

```diff
@@ -89,61 +87,59 @@ library Scalar {
             // data
             o := mload(p)
         }
-        result = FE.wrap(o);
+        result = o;
```

Comment:
We can remove the `o` variable and directly using `result`.

Reply:
done! thanks!!!!